### DISCUSSION TITLE
2024/08/11/%E6%B5%8B%E8%AF%95%E6%96%87%E7%AB%A0/

### DISCUSSION THREAD

**Discussion by giscus** [MengZe2l/mengze2l.github.io#2] [General]
# 2024/08/11/%E6%B5%8B%E8%AF%95%E6%96%87%E7%AB%A0/

1234546 126494

http://localhost:4000/2024/08/11/%E6%B5%8B%E8%AF%95%E6%96%87%E7%AB%A0/

**Comment by MengZe2l** [comment]
## 123
测试
```js
js = "javascript"
```
